### PR TITLE
Feature/nuxt3

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: components/Launcher.vue [hunk 1]
```diff
@@ -1,93 +1,63 @@
...
+onMounted(() => {
+  if (process.client) {
+    const config = useRuntimeConfig()
+    if (config.public.NODE_ENV !== 'production') {
+      cloud_store.$patch({ is_captcha_validated: true })
     }
-  },
+  }
+})
+onUnmounted(() => { console.log("unmount") })
```
Comment (BotellaA) [inline components/Launcher.vue:49]: to remove ?

File: components/Launcher.vue [hunk 2]
```diff
@@ -1,93 +1,63 @@
...
-          await this.$recaptcha.reset()
-        } catch (error) {
-          console.log('Login error:', error)
-        }
-    },
-  },
+async function submit_recaptcha (token) {
+  try {
+    const config = useRuntimeConfig()
+    console.log('ReCaptcha token:', token)
```
Comment (BotellaA) [inline components/Launcher.vue:54]: Remove all the console.log?

File: components/RemoteRenderingView.client.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  connect()
+  resize()
+
+})
+
+onBeforeUnmount(() => {
+  if (this.subscription) {
+    this.subscription.unsubscribe()
+    this.subscription = null
+  }
```
Comment (BotellaA) [inline components/RemoteRenderingView.client.vue:56]: to remove

File: components/RemoteRenderingView.client.vue [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+<style>
+.viewer {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 500px;
+  overflow: hidden;
+  z-index: 1;
+}
```
Comment (BotellaA) [inline components/RemoteRenderingView.client.vue:85]: Is it still useful?

File: components/SideMenu.vue [hunk 5]
```diff
@@ -1,117 +1,99 @@
...
-      message: '',
-      DataFilename: '',
-      TextureFilename: '',
-      items: [
-        {
-          file: []
-          , messages: "Please select a data file"
+const cloud_store = use_cloud_store()
+const ws_link_store = use_ws_link_store()
+const vtk_store = use_vtk_store()
```
Comment (BotellaA) [inline components/SideMenu.vue:37]: unused?

File: components/SideMenu.vue [hunk 6]
```diff
@@ -1,117 +1,99 @@
...
+    }
+    if (current_item.file.length) {
+      reader.readAsDataURL(current_item.file[0])
     }
   }
 }
+
+async function load (DataFilename) {
+  console.log('load')
+  console.log('DataFilename', DataFilename)
```
Comment (BotellaA) [inline components/SideMenu.vue:95]: remove as many debug logger as possible

File: nuxt.config.js [hunk 7]
```diff
@@ -1,84 +1,93 @@
...
-  },
+  // ** Global CSS
+  css: ['vuetify/lib/styles/main.sass'],
 
-  // Axios module configuration: https://go.nuxtjs.dev/config-axios
-  axios: {},
+  // ** Nuxt.js modules
+  modules:
+    [
+      // ['@dargmuesli/nuxt-cookie-control',
```
Comment (BotellaA) [inline nuxt.config.js:39]: To uncomment?

File: nuxt.config.js [hunk 8]
```diff
@@ -1,84 +1,93 @@
...
   },
 
-  // Build Configuration: https://go.nuxtjs.dev/config-build
+  // ** Build configuration
   build: {
+    transpile: ['vuetify'],
+  },
+  vue: {
+    compilerOptions: {
+      isCustomElement: (tag) => ['md-linedivider'].includes(tag)
```
Comment (BotellaA) [inline nuxt.config.js:87]: Where is this used?

File: stores/ws_link.js [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+        .connect(config)
+        .then((validClient) => {
+          connectImageStream(validClient.getConnection().getSession());
+          this.client = validClient
+          clientToConnect.endBusy();
+
+          // Now that the client is ready let's setup the server for us
+          this.ws_initialize_server()
+          this.client.getRemote().vtk.reset().catch(console.error);
+          console.log("created ", this.is_client_created);
```
Comment (BotellaA) [inline stores/ws_link.js:81]: some cleaning in this file too?

**Review by BotellaA — APPROVED** [review]

File: nuxt.config.js (hunk 8 above)
Comment (JulienChampagnol) [reply nuxt.config.js:87]: Nuxt is having trouble identifying which component is local and which is from vuetify, so this is used for every vuetify component

File: components/SideMenu.vue [hunk 10]
```diff
@@ -1,117 +1,99 @@
...
+  vtk_store.send_filenames({ DataFilename })
```
Comment (JulienChampagnol) [inline components/SideMenu.vue:96]: vtk_store is used here

**Pull request merged**